### ISSUE TITLE
dbus sending only works with given number/account

### ISSUE THREAD

**Issue by fettboy** [AsamK/signal-cli#847]
Hello,
I have the problem that I can´t send with dbus if not giving the sender-number.
I tried the following:

1.)Sending with number works successfully directly from signal-cli  
     signal-cli --config /var/lib/signal-cli  send -m "Message direct from signal-cli" +49xxxxxxxxxxx

2.) Starting the dbus-service and trying WITH sender-number works
     systemctl start signal-cli
     su signal-cli
     signal-cli --dbus-system --config /var/lib/signal-cli -u +49yyyyyyyyyyyy send +49xxxxxxxxx -m "Signal-CLI via DBus works:-)"

3.) Starting the dbus-service and trying WITHOUT sender-number ERRORS:
     systemctl start signal-cli
     su signal-cli
     signal-cli --dbus-system --config /var/lib/signal-cli  send +49xxxxxxxxx -m "Signal-CLI via DBus .... :-("
     > The method `org.asamk.Signal.getSelfNumber' does not exist on this object.

4.) Checked: sender number is registered 
    (run as user signal-cli)
     signal-cli --config /var/lib/signal-cli  listIdentities
      +49yyyyyyyyyy: TRUSTED_VERIFIED Added: Fri Dec 24 11:24:33 CET 2021 Fingerprint: 05 ....e 75 38 Safety Number: 644..     
      +49xxxxxxxxxx: TRUSTED_VERIFIED Added: Fri Dec 24 11:26:36 CET 2021 Fingerprint: 05 ... bd dd 7c Safety Number: 119..

used version of signal-cli: v.0.9.2

I need a hint, what to do to make step 3 (giving config, but no user-number) working. 
Thanks in advance.

**Comment by AsamK** [comment]
Currently either the daemon or the send command must be started with the username flag. So for step 3 to work you need to start signal-cli with `-u +49yyyyyyyyyyyy daemon`

**Comment by fettboy** [comment]
Thank you for the hint! With it I found that my systemd-config executed the command without that option.
I took the default-systemd-config from repository before, may be it is better to change the default to this?
/etc/systemd/system/signal-cli.service:
    ...
    ExecStart=/usr/local/bin/signal-cli --config /var/lib/signal-cli  -u +49yyyyyyyyyyyy daemon --system
    ...

[
  I also stumbled over the option --session and --system, because of the similarity in names (look closer fettboy ;-) )
]
Now, it runs perfect, Thank you!
